is_iam_enabled.go

```diff
@@ -0,0 +1,76 @@
+package alks
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"strings"
+)
+
+type IsIamRoleRequest struct {
```

Comment:
I know this is a bit of bike shedding, but I feel like these structure names do not match the their purpose very well.
```suggestion
type IsIamEnabledRequest struct {
```